src/components/surveys/survey-design/question-edit/rhs-subcontrols/Time.tsx

```diff
@@ -18,27 +18,29 @@ import {theme} from '@style/theme'
 import {FormatOptionsTime, TimeQuestion} from '@typedefs/surveys'
 import React from 'react'
 
-const Labels = styled('div', {label: 'labels'})(({theme}) => ({
-  backgroundColor: '#fff',
-  padding: theme.spacing(2, 1.5),
-  marginTop: theme.spacing(2),
+// TODO: syoung 10/11/2023 Revisit this design later - it doesn't really make sense to allow time questions to constrain to future/past.
```

Comment:
Not following you here--why not? "What year did you quit smoking" = constrained to past, "When is your next scheduled appointment" = constrained to future, e.g.

Reply:
Yeah, no. This is for "what time of day". 

Here's what it looks like now:

<img width="277" alt="Screenshot 2023-10-11 at 9 17 40 PM" src="https://github.com/BridgeDigitalHealth/open-bridge-web/assets/5649217/3cdd802a-8e7c-4a74-a443-acb81c646e2d">

Here's what the designs are asking for:

<img width="265" alt="Screenshot 2023-10-11 at 9 20 56 PM" src="https://github.com/BridgeDigitalHealth/open-bridge-web/assets/5649217/5a88b205-b469-4c87-89b6-7238304902cf">

Neither of which make sense, really. My guess is that Alina realized that, started to change it to use the UI/UX for numeric values and then forgot about it.